src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const males = [...people].filter(person => (
+    century
+      ? (Math.ceil(person.died / 100) === century && person.sex === 'm')
+      : (person.sex === 'm')
+  ));
+
+  return Math.round(males.map(man => man.died - man.born)
+    .reduce((man1, man2) => man1 + man2) / males.length * 100) / 100;
```

Comment:
Actually first arg of the `reduce` cb is not the taken value but the accumulator
Proper naming helps to keep your code simpler
```suggestion
    .reduce((acc, age) => acc + age) / males.length * 100) / 100;
```